### ISSUE TITLE
[Feature Request] Attach to process

### ISSUE THREAD

**Issue by Spidy123222** [libimobiledevice/libimobiledevice#1128]
Hi I am spidy123222 and I am requesting this feature because I heard of a method that can attach a process when it launches the app and having to use remote debugserver seems less reliable and would like to have "Attach To Process". This would be debugging a app I would be using often called UTM and DolphinIOS where it uses a debugger to do JIT on the app. It has been found that on Xcode you can attach to process and make it able to work without being tethered via a cable and it being more reliable than traditional remote debugserver.

This is their tutorial by the way. ![image](https://user-images.githubusercontent.com/64176728/110730299-14af9300-81d5-11eb-8f66-1a9f22b88802.jpeg)

This can be used on a raspberry pi and on windows if it's on libimobiledevice.

**Comment by jkcoxson** [comment]
tldr; we want to attach the debugger to a PID, like you can in Xcode 

Looking at idevicedebug it looks like this could be possible to contact the iOS debugger for a specific process, if you could point us in the right direction or to some documentation that would be great!

**Comment by Spidy123222** [comment]
From going through it seems you can use udid to activate debugger and attach it to app bundle. But doing it over WiFi ain't supported it seems on Linux.
